cmd/ipfs-cluster-ctl/main.go

```diff
@@ -277,6 +277,10 @@ If you prefer faster adding, add directly to the local IPFS and trigger a
 					Name:  "quieter, Q",
 					Usage: "Write only final hash to output",
```

Comment:
Nothing to do with this PR, but this description is wrong. `quieter` doesn't print just hash. It also prints name and size(that's the behaviour I see in master). It's a little that I think we can push along with this PR.

Reply:
It does just print the hash? Unless you mean in the json output, to which this does not really apply because we print the json objects.

```
[18:30:23] ~/ipfstest $ ipfs-cluster-ctl  add -Q -r testsh/
QmekoCaPuQ8wcWhuws4KRZnNhzaTNrbM1e6dqgWRVLyZnw
[18:30:28] ~/ipfstest $ ipfs-cluster-ctl  add --no-stream -Q -r testsh/
QmekoCaPuQ8wcWhuws4KRZnNhzaTNrbM1e6dqgWRVLyZnw
```

Reply:
yup, I was talking about the json output

Reply:
yeah, json output should print valid json in any case. -q and -Q have text-formatting implications which do not apply to json (here or in `ipfs add`)